tests/metaworld/envs/mujoco/sawyer_xyz/test_scripted_policies.py

```diff
@@ -28,7 +32,9 @@
     ['assembly-v1', SawyerAssemblyV1Policy(), .0, 1.],
     ['basketball-v1', SawyerBasketballV1Policy(), .0, .98],
     ['button-press-topdown-v1', SawyerButtonPressTopdownV1Policy(), .0, 1.],
+    ['button-press-topdown-wall-v1', SawyerButtonPressTopdownWallV1Policy(), .0, 1.],
     ['button-press-v1', SawyerButtonPressV1Policy(), .0, .93],
```

Comment:
why does ButtonPress get 93% when the harder variants (Wall and TopdownWall) get 100%?

Reply:
Pressing from the top is always easier than pressing from the side, and the wall in TopdownWall doesn't actually get in the way of anything (it's basically at the same height as the button). But you're right that of the two that come from the side, the non-wall variant should do better. I'll work on it.